### ISSUE TITLE
Custom Workspaces (not just the standard 1-10)

### ISSUE THREAD

**Issue by jontstaz** [Aylur/ags#230]
Hi,

I'm coming from eww and I have pretty much setup AGS how I had it in eww apart from some of my workspaces. I use Hyprland and I have 2 monitors. My main monitor is on the right and my secondary is on the left. In Hyprland I set custom workspaces and each workspace is linked to a monitor. So ESC (Main/Default), 1-10 and G (Games) will all open on my main monitor. I then have Z, X and C (named web, web2 and web3) which always open on my secondary monitor. In eww I was able to detect all active workspaces in my top-bar using a bash script that called `hyprctl workspaces`, and then subsequently called `eww update ...` to update the list of open workspaces. 

Anyway, long story short: is it possible to do something similar in ags? Currently my top-bar just has my workspaces numbered 1-10 and if I'm currently using a workspace not in 1-10, then the ags top-bar just displays as if there is no currently active workspace.

**Comment by Aylur** [comment]
you can of course set it up however you want, checkout the [Hyprland service wiki page](https://github.com/Aylur/ags/wiki/Hyprland) that should have all the information you need

**Comment by midn8hustlr** [comment]
I have implemented similar thing where workspace is not limited to 10. Check [this](https://github.com/end-4/dots-hyprland/pull/209#issue-2103833045). You can copy the workspace code from this repo or use these dotfiles.
